### PR TITLE
feature: Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by Guilherme-Vasconcelos**

### CODE REVIEW

**Comment by strager** [comment]
> error: string literal of length 73968 exceeds maximum length 65536 that C++ compilers are required to support [-Werror,-Woverlength-strings]

Welp, I guess we need to disable this warning. 😆 Please remove it in `.github/workflows/build-and-test.yml`.

File: po/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,1657 @@
+# Code generated by tools/update-translator-sources. DO NOT EDIT.
+# source: (various)
```
Comment (strager) [inline po/pt_BR.po:2]: Must fix: This comment shouldn't exist. (I should update the instructions to remove this message...)

**Review by strager — CHANGES_REQUESTED** [review]

**Comment by Guilherme-Vasconcelos** [comment]
> error: string literal of length 73968 exceeds maximum length 65536 that C++ compilers are required to support [-Werror,-Woverlength-strings]
> 
> Welp, I guess we need to disable this warning. :laughing: Please remove it in `.github/workflows/build-and-test.yml`.

Fixed! By the way, do you know why GCC wasn't giving the same warning? Looks like both compilers had that flag, but only clang was failing

**Comment by strager** [comment]
> do you know why GCC wasn't giving the same warning? Looks like both compilers had that flag, but only clang was failing

I experimented, and it seems that GCC only warns in C code, not in C++ code.

**Review by strager — APPROVED** [review]

**Pull request merged**